stores/GenericSensors/GenericSensorsService.ts

```diff
@@ -0,0 +1,23 @@
+export const GenericSensorsService = {
```

Comment:
Please add a comment above (just for consistency with the other services)

Reply:
done